TrackingTools/TrajectoryFiltering/interface/SeedExtensionTrajectoryFilter.h

```diff
@@ -0,0 +1,51 @@
+#ifndef SeedExtensionTrajectoryFilter_H
+#define SeedExtensionTrajectoryFilter_H
+
+#include "TrackingTools/TrajectoryFiltering/interface/TrajectoryFilter.h"
+
+class SeedExtensionTrajectoryFilter final : public TrajectoryFilter {
+public:
+
+  explicit SeedExtensionTrajectoryFilter() {} 
+  
+  explicit SeedExtensionTrajectoryFilter(edm::ParameterSet const & pset, edm::ConsumesCollector&) :
+     theStrict(pset.existsAs<bool>("strictSeedExtension") ? pset.getParameter<bool>("strictSeedExtension"):false),
```

Comment:
it is a PSet...
@makortel looked into this kind of issue last year, he may comment further (there is also an issue with PSet_Ref)
In any case with @mtosi we will review the use in HLT and eventually remove all those "exitsAs" that I dislike as well.
At the moment we need them if we with to progress fast (as HLT asked)